lib/ui/product/widget/product_description.dart

```diff
@@ -0,0 +1,336 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_icons/flutter_icons.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:flipper/utils/HexColor.dart';
+
+class ProductDescription extends StatefulWidget {
+  const ProductDescription({Key key}) : super(key: key);
+
+  @override
+  _ProductDescriptionState createState() => _ProductDescriptionState();
+}
+
+class _ProductDescriptionState extends State<ProductDescription> {
+  double _checked;
+  TextEditingController quantityController = new TextEditingController();
```

Comment:
These variables should be in ViewModel for using this https://github.com/FilledStacks/stacked